paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetViewModelTest.kt

```diff
@@ -2082,20 +2061,18 @@ class CustomerSheetViewModelTest {
         viewModel.viewState.test {
             assertThat(awaitItem()).isInstanceOf<AddPaymentMethod>()
 
-            viewModel.handleViewAction(
-                CustomerSheetViewAction.OnConfirmUSBankAccount(usBankAccount)
-            )
```

Comment:
Replacing the single `OnConfirmUSBankAccount` with two actions: selecting the bank account and confirming via the primary button.